packages/database/src/migrations/20210804063133-AddFijiCOVIDDashboardTestsByDay-modifies-data.js

```diff
@@ -0,0 +1,145 @@
+'use strict';
+
+import {
+  insertObject,
+  generateId,
+  findSingleRecord,
+  findSingleRecordBySql,
+  deleteObject,
+} from '../utilities';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+const generateConfig = () => ({
+  code: `FJ_Covid_Tests_By_Day`,
+  reportConfig: {
+    fetch: {
+      aggregations: [
+        {
+          type: 'RAW',
+          config: {
+            dataSourceEntityType: 'facility',
+          },
+        },
+      ],
+      dataElements: ['COVIDTest_FJNumTest'],
+    },
+    transform: [
+      {
+        '...': 'drop',
+        value: 'sum',
+        period: 'group',
+        transform: 'aggregate',
+      },
+      {
+        "'name'": "periodToDisplayString($row.period, 'DAY')",
+        transform: 'select',
+        "'timestamp'": 'periodToTimestamp($row.period)',
+        "'Tests'": '$row.value',
+        '...': [],
```

Comment:
```suggestion
```
Think I mentioned this in another PR, but this field is optional 👍

Reply:
Amazing